tests/containers/apparmor.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: apparmor-utils
+# Summary: Test apparmor utilities
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use services::apparmor;
+use utils;
+
+sub is_enabled {
+    # according to Linux' Documentation/admin-guide/LSM/index.rst,
+    # the current security modules can be found in /sys/kernel/security/lsm
+    return script_run 'grep apparmor /sys/kernel/security/lsm' == 0;
```

Comment:
parenthesis definitely needed over here

```
Argument "grep apparmor /sys/kernel/security/lsm" isn't numeric in numeric eq (==) at /var/lib/openqa/pool/14/os-autoinst-distri-opensuse/tests/containers/apparmor.pm line 19.
	apparmor::is_enabled() called at /var/lib/openqa/pool/14/os-autoinst-distri-opensuse/tests/containers/apparmor.pm line 50
	apparmor::run(apparmor=HASH(0x55da98319960)) called at /usr/lib/os-autoinst/basetest.pm line 352
	eval {...} called at /usr/lib/os-autoinst/basetest.pm line 346
	basetest::runtest(apparmor=HASH(0x55da98319960)) called at /usr/lib/os-autoinst/autotest.pm line 412
```

Reply:
Fixed in [398823](https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/3988234a5118e4f6ef36386634d858b9d1c148a5).